nomos-da/kzgrs/Cargo.toml

```diff
@@ -39,7 +38,7 @@ name = "fk20"
 harness = false
 
 [features]
-default = ["single"]
+default = ["single", "rayon"]
```

Comment:
To enable `rayon` support please use `parallel` feature, default for now should stay as is.